var/spack/repos/builtin/packages/gdb/package.py

```diff
@@ -108,6 +111,9 @@ def configure_args(self):
         if "+tui" in self.spec:
             args.append("--enable-tui")
 
+        if "+debuginfod" in self.spec:
+            args.append("--with-debuginfod")
```

Comment:
FYI.  You could be using helper functions for these (https://spack.readthedocs.io/en/latest/build_systems/autotoolspackage.html#with-or-without).

Reply:
I adopted them in most cases, but left it for python and gmp because these dependencies require more complex flags